### PR TITLE
chore: Bump Polkadot api

### PR DESCRIPTION
**Pull request by Tbaut**
In an attempt to solve #525 still in vain.
Bumping polkadot packages can't hurt still.

I was able to sign a tx, please make sure all tests pass (I still can't run them).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,11 +38,11 @@
   "dependencies": {
     "@plugnet/util": "^1.1.100",
     "@plugnet/wasm-crypto-js": "^0.11.102",
-    "@polkadot/api": "^0.98.0-beta.0",
-    "@polkadot/reactnative-identicon": "^0.46.1",
-    "@polkadot/types": "^0.98.0-beta.1",
-    "@polkadot/util": "^1.7.1",
-    "@polkadot/util-crypto": "1.7.1",
+    "@polkadot/api": "^0.100.1",
```
Comment (yjkimjunior) [inline package.json]: i think it's `1.0.0-beta.7`

it seems `yarn upgrade-interactive --latest` ignores the beta tags

File: package.json [hunk 2]
```diff
@@ -38,11 +38,11 @@
...
+    "@polkadot/reactnative-identicon": "^0.47.1",
+    "@polkadot/types": "^0.100.1",
+    "@polkadot/util": "^1.8.1",
+    "@polkadot/util-crypto": "^1.8.1",
```
Comment (yjkimjunior) [inline package.json]: same here `2.0.0-beta.4` https://github.com/polkadot-js/common/tree/master/packages

File: package.json [hunk 3]
```diff
@@ -38,11 +38,11 @@
...
     "@plugnet/wasm-crypto-js": "^0.11.102",
-    "@polkadot/api": "^0.98.0-beta.0",
-    "@polkadot/reactnative-identicon": "^0.46.1",
-    "@polkadot/types": "^0.98.0-beta.1",
-    "@polkadot/util": "^1.7.1",
-    "@polkadot/util-crypto": "1.7.1",
+    "@polkadot/api": "^0.100.1",
+    "@polkadot/reactnative-identicon": "^0.47.1",
+    "@polkadot/types": "^0.100.1",
+    "@polkadot/util": "^1.8.1",
```
Comment (yjkimjunior) [inline package.json]: same here 2.0.0-beta.4 https://github.com/polkadot-js/common/tree/master/packages

File: package.json (hunk 1 above)
Comment (yjkimjunior) [reply package.json]: https://github.com/polkadot-js/api/blob/be258a1f7c997ad8e31e8b515833c943672bfb76/package.json#L2

**Comment by Tbaut** [comment]
Do we want the beta though? I did a `yarn upgrade --latest` on those specific packages. We used to have the beta only for `/api` and `/types`, not sure what `beta` would bring or break.

**Comment by Tbaut** [comment]
Chatted with Jaco, `beta` is needed if we want to support everything indeed.

**Comment by Tbaut** [comment]
I was able to do a tx with beta as well with `kusama --dev`.

**Pull request merged**